src/coreclr/debug/daccess/request.cpp

```diff
@@ -435,7 +435,7 @@ ClrDataAccess::GetMethodTableSlot(CLRDATA_ADDRESS mt, unsigned int slot, CLRDATA
             MethodDesc * pMD = it.GetMethodDesc();
             if (pMD->GetSlot() == slot)
             {
-                *value = pMD->GetMethodEntryPoint();
+                *value = pMD->GetMethodEntryPoint_NoAlloc();
```

Comment:
Is `NULL` a valid return here?

Reply:
Returning NULL is going to break the method enumeration done by DumpMT. Related discussion: https://github.com/dotnet/diagnostics/pull/4760#discussion_r1653807525 . We may want to introduce a better method enumeration DAC API.